1-quality.rst

```diff
@@ -9,11 +9,16 @@ Make sure you've got the PROJECT location defined, and your data is there:
 
    set -u
    printf "\nMy raw data is in $PROJECT/data/, and consists of $(ls -1 ${PROJECT}/data/*.fastq.gz | wc -l) files\n\n"
+   set +u
```

Comment:
Any reason we want to allow unbound variables? Might be better to proceed in "safe mode". \*shrug\*